doc/users/interactive.rst

```diff
@@ -146,45 +139,43 @@ Interactive mode controls:
 
 - whether created figures are automatically shown
 - whether changes to artists automatically trigger re-drawing existing figures
-- whether `.pyplot.show` returns immediately or after all of the
-  figures have been closed when given no arguments
-
+- when `.pyplot.show` exits: immediately, or after all of the figures have been closed if given no arguments
```

Comment:
I think that "returns" is the correct language here.  Processes exit, functions return.

Reply:
I hate both and think neither is super clear, especially here where it's not super clear that `pyplot.show` is being talked about in the context of it being a function.

Reply:
"exits" is definitely incorrect for a function.  Perhaps `pyplot.show()` (with parentheses) would make things clearer?